pyspecdata/DCCT_function.py

```diff
@@ -402,20 +422,20 @@ def place_labels(
                 ha="right",
                 rotation=45,
                 clip_on=False,
-                transform=(
-                    IdentityTransform()
-                    + ScaledTranslation(ax_x, ax_y, fig.transFigure)
-                ),
+                transform=arrow_trans,
                 color="k",
             )
             if check_for_label_num:
                 label_placed[this_label_num] = 1
 
+    # }}}
+    label_placed = np.zeros(num_dims)
```

Comment:
This was above randomly between the 2 function definitions so I moved it here for organization